doc/man/restic-backup.1

```diff
@@ -91,7 +91,7 @@ Exit status is 3 if some source data could not be read (incomplete snapshot crea
 
 .PP
 \fB\-\-parent\fP=""
-	use this parent \fB\fCsnapshot\fR (default: last snapshot in the repo that has the same target files/directories)
+	use this parent \fB\fCsnapshot\fR (default: last snapshot in the repo that has the same target files/directories, and is not newer than the snapshot time)
```

Comment:
The man pages are regenerated automatically before a new release. Please remove this change from the PR.

Reply:
Done, pushed a commit to add a note to CONTRIBUTING.md as well to document this.

Reply:
Good idea. Thanks.